reg/correctMotion_DFT.m

```diff
@@ -0,0 +1,64 @@
+function correctMotion_DFT(hdf5_dir,hdf5_in_name,hdf5_out_name)
+
+dataset = '/Data/Images';
+
+h5att = h5info(fullfile(hdf5_dir,hdf5_in_name),dataset);
+imageStackSize = h5att.Dataspace.Size;
+rows = imageStackSize(1);
+cols = imageStackSize(2);
+frames = imageStackSize(3);
```

Comment:
FYI, in the `kimth/f2f-reg` branch, there is a helper function called [`get_dataset_info`](https://github.com/inanhkn/analysis/blob/kimth/f2f-reg/io/get_dataset_info.m) that performs the above fiew lines.